content/en/docs/tasks/administer-cluster/use-cascading-deletion.md

```diff
@@ -79,20 +80,20 @@ kubectl delete deployment nginx-deployment --cascade=foreground
    The output contains a `foregroundDeletion` {{<glossary_tooltip text="finalizer" term_id="finalizer">}}
    like this:
 
-   ```
-    "kind": "Deployment",
-    "apiVersion": "apps/v1",
-    "metadata": {
-        "name": "nginx-deployment",
-        "namespace": "default",
-        "uid": "d1ce1b02-cae8-4288-8a53-30e84d8fa505",
-        "resourceVersion": "1363097",
-        "creationTimestamp": "2021-07-08T20:24:37Z",
-        "deletionTimestamp": "2021-07-08T20:27:39Z",
-        "finalizers": [
-          "foregroundDeletion"
-        ]
-        ...
+   ```json
```

Comment:
(nit)
Given that there is `...` in the example, I would highlight this as YAML.
In my experience, the YAML highlighter handles that case better.

Also, all valid JSON is valid YAML.

Reply:
nit: Check lines 96 - 117?

Reply:
Alright, this segment is part of command output, so the syntax highlighting can be turned off. I'm removing the "json" syntax tag.